.github/workflows/main.yaml

```diff
@@ -0,0 +1,38 @@
+name: Docker Image CI
```

Comment:
Looks great. I think we have a lot of options for wrapping this up.

Reply:
Thank you. After the initial release is ready, we can limit this action to run only on the `master` branch.